.github/workflows/ci_windows.yaml

```diff
@@ -47,16 +48,32 @@ jobs:
           vcpkgDirectory: C:/vcpkg
           doNotUpdateVcpkg: true  # the preinstalled vcpkg is updated regularly
 
-      - name: Install dependencies from vcpkg
+      - name: Install dependencies with vcpkg
         run: |
+          # create our own triplet
+          New-Item `
+            -Force `
+            -ItemType File `
+            -Path "${{ env.vpkg_triplet_path }}/x64-windows-static-md-release.cmake"
+          Add-Content `
+            -Path "${{ env.vpkg_triplet_path }}/x64-windows-static-md-release.cmake" `
+            -Value @("set(VCPKG_TARGET_ARCHITECTURE x64)",
+              "set(VCPKG_LIBRARY_LINKAGE static)",
```

Comment:
Maybe consider dynamic linking?

Reply:
IMO dynamic linking libraries is useless.

Reply:
> IMO dynamic linking libraries is useless.

Really so useless? At one time, I switched to using dynamic libraries, as this significantly reduced the linking time of application executable.

Reply:
>Really so useless? 

Not so useless if the libraries could be shared but it is not the case here.

>At one time, I switched to using dynamic libraries, as this significantly reduced the linking time of application executable.

Maybe you enabled [LTCG](https://learn.microsoft.com/en-us/cpp/build/reference/ltcg-link-time-code-generation?view=msvc-170)? That is quite slow and CI doesn't use it.
It seems it doesn't take a long time for CI to link: https://pipelinesghubeus23.actions.githubusercontent.com/ssH9PGsbkWUcjTkWZltMgGgVgcDqZbWeI5I5izohk2YszdiJwj/_apis/pipelines/1/runs/28452/signedlogcontent/2?urlExpires=2023-11-18T10%3A42%3A01.7212931Z&urlSigningMethod=HMACV1&urlSignature=ywvwKQw8SA5lGbaXR3uZcS0LeVF767AgIvyXu4q%2FpDY%3D (search for `Linking CXX executable qbittorrent.exe`) It is about 1 or 2 seconds at max.
I will consider dynamic linking if it is really slow.

Reply:
Of course, if it is not possible to save once built libraries to use them again, then it probably does not matter much.

Reply:
> Maybe you enabled [LTCG](https://learn.microsoft.com/en-us/cpp/build/reference/ltcg-link-time-code-generation?view=msvc-170)? That is quite slow and CI doesn't use it.

I do not remember. Perhaps it was. I used some suggested/recommended/predefined flags.